user_flair/new_hero_flair.scss

```diff
@@ -0,0 +1,198 @@
+.flair-hero::before {
+  background: url(%%flair-hero%%);
+}
+
+.flair-h-Abathur::before{
```

Comment:
In my opinion, class names should be kept all lowercase. I don't think we'll run into any collision issues by not capitalizing hero names. What do you think?